manifests/cloud-config/scripts/generate-cloud-config.sh

```diff
@@ -10,10 +10,6 @@ for i in "${PAAS_CF_DIR}"/manifests/cloud-config/operations.d/*.yml; do
   opsfile_args+="-o $i "
 done
 
-if [ "$AWS_ACCOUNT" = "dev" ]; then
-  opsfile_args+="-o ${PAAS_CF_DIR}/manifests/cloud-config/operations/use-spot-instances-in-dev.yml"
```

Comment:
This means we can [delete that file](https://github.com/alphagov/paas-cf/blob/main/manifests/cloud-config/operations/use-spot-instances-in-dev.yml) and the [variables for spot prices ](https://github.com/alphagov/paas-cf/blob/main/manifests/variables.yml#L8) as well, unless it's used somewhere else?

Reply:
I ran it into dev03. 

I was wondering whether to leave everything else in place just in case the estimation of costs in the ticket was wildly out and spots had to be re-implemented quickly. On reflection, I'm going with your suggestion.